packages/lib/gpcircuits/scripts/parameters/paramGen.json

```diff
@@ -27,27 +27,79 @@
   },
   {
     "maxObjects": 1,
-    "maxEntries": 5,
+    "maxEntries": 11,
     "merkleMaxDepth": 5,
     "maxNumericValues": 0,
     "maxEntryInequalities": 0,
     "maxLists": 1,
-    "maxListElements": 200,
+    "maxListElements": 500,
```

Comment:
500 seems very big.  I'm all for having at least one circuit with a big list, but I think most of them shouldn't.
In particular, I think we need some circuits with more than 5 entries (e.g. for ZuKyc, or for tickets with all fields revealed), but which have no list, or a small list (5-20 entries) which will come up frequently.